products/surface.md

```diff
@@ -168,7 +223,7 @@ releases:
 
 -   releaseCycle: Surface 2
     releaseDate: 2013-10-22
-    eol: 2018-04-10
+    eol: 2023-01-10
```

Comment:
wrong date , it is already EOLed : https://support.microsoft.com/en-us/surface/surface-2-update-history-caec7bb3-4c17-f7e2-d4ab-071d3d3c2812

Reply:
Fixed.